metadata.json

```
@@ -26,7 +26,7 @@
     },
     {
       "name": "camptocamp/systemd",
-      "version_requirement": ">= 0.4.0 < 3.0.0"
+      "version_requirement": ">= 0.4.0 <= 3.0.0"
```

I think we want to change the minimum version here:
```suggestion
      "version_requirement": ">= 3.0.0 < 4.0.0"
```